test/API/tattr.c

```diff
@@ -6516,18 +6506,20 @@ test_attr_delete_by_idx(hbool_t new_format, hid_t fcpl, hid_t fapl)
                                         "Creation Order Index\n"))
                     } /* end if */
                     else {
-#ifndef NO_DECREASING_ALPHA_ITER_ORDER
-                        if (use_index)
-                            MESSAGE(5, ("Testing Deleting Attribute By Name Index in Decreasing Order "
-                                        "w/Creation Order Index\n"))
-                        else
-                            MESSAGE(5, ("Testing Deleting Attribute By Name Index in Decreasing Order w/o "
-                                        "Creation Order Index\n"))
-#else
-                        continue;
-#endif
-                    } /* end else */
-                }     /* end else */
+                        if (vol_cap_flags_g & H5VL_CAP_FLAG_CREATION_ORDER) {
+                            if (use_index)
```

Comment:
In this case and the other cases like this in this file, the check for the creation order flag should be moved inside the `if (use_index)` block since a connector could support Name Order without supporting Creation Order.

Reply:
Got it, I can do that.